hooks/post_plan.py

```diff
@@ -186,7 +187,7 @@ def _validate_no_changes_when_blue_green_deployment_enabled(self) -> None:
         )
         if resource_changes:
             self.errors.append(
-                f"No changes allowed when Blue/Green Deployment enabled, detected changes: {resource_changes}"
+                f"No changes allowed when sync after Blue/Green Deployment completed, detected changes: {resource_changes}"
```

Comment:
This message is hard to understand. 

Maybe something like: 
```suggestion
                f"There are pending resource changes after a Blue/Green Deployment.  This should not happen. Detected changes: {resource_changes}"
```